content/docs/command-reference/exp/index.md

```diff
@@ -0,0 +1,44 @@
+# exp[eriments]
+
+A set of commands to generate and compare local _experiments_:
+[show](/doc/command-reference/exp/show),
+[checkout](/doc/command-reference/exp/show),
+[diff](/doc/command-reference/exp/diff), [run](/doc/command-reference/exp/run),
+[resume](/doc/command-reference/exp/resume), and
+[gc](/doc/command-reference/exp/gc).
+
+## Synopsis
+
+```usage
+usage: dvc exp[eriments] [-h] [-q | -v] {show,checkout,diff,run,resume,res,gc} ...
+
+positional arguments:
+  COMMAND
+    show                Print experiments.
+    checkout            Checkout experiments.
+    diff                Show changes between experiments in the DVC repository.
+    run                 Reproduce complete or partial experiment pipelines.
+    resume (res)        Resume checkpoint experiments.
+    gc                  Garbage collect unneeded experiments.
+```
+
+## Types of experiments
+
+DVC supports two types of experiments:
+
+1. Standalone experiments
```

Comment:
Link to https://dvc.org/doc/start/experiments ?

Reply:
Ah no, I see you mean something else here. But still we should link to/from that page, probably. Adding the relevant notes. WDYT @pmrowla ?

Reply:
Or did you mean that "std exps" are the old ones (one per commit) and "checkpoints" are the ones `dvc exp` uses?

Reply:
p.s. Oh, and if not, let's make this **Types** section part of the description, please — may not even need a header. It only exists in metrics and plots refs. to redirect users to one or the other before they read an entire desc. for the wrong one, but there's no such ambiguity here.

Reply:
In this case standalone is still the "new" ones, just without checkpoint outputs. In this case I'm referring to using `dvc exp` to generate lots of one-per-commit/branch experiments with modified parameters and/or code, but without needing to commit them in the main repo.

Reply:
OK. I'm not sure we want to call checkpoints a different type of experiment. Maybe it's a feature on top of experiments? I need to play with this more to be able to tell...

Reply:
But this should def be in the **Description** section.